internal/controller/controller.go

```diff
@@ -8,9 +8,12 @@ type ControllerInterface interface {
 	Init(masterHost string)
 	Login()
 	Environment()
+	EnvironmentAccess(envId ...string)
 	Version()
 	Services(envId ...string)
-	Teleport(teleportAllFlag bool)
+	ServiceAccess(envId string,blockId string)
```

Comment:
what is the diff between `Access` and `ServiceAccess`? maybe we can rename them to differentiate what they do?